ansible/roles/schulcloud-server-core/tasks/main.yml

```diff
@@ -119,6 +125,25 @@
       template: api-files-ingress.yml.j2
       apply: yes
 
+  - name: EduSharingDeployment
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: api-edu-sharing-deployment.yml.j2
+
+  - name: EduSharingDeployment
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: api-edu-sharing-deployment.yml.j2
```

Comment:
Doppelt

Reply:
Fixed by removing duplicated code!

Reply:
@mamutmk5 The code for `FileStorageDeployment` is also available twice. Is this intended?

Reply:
@bergatco no it was not intended to have the code for the `FileStorageDeployment` double